linkerd/stack/src/new_service.rs

```diff
@@ -21,6 +23,20 @@ pub struct FromMakeService<S> {
 #[derive(Clone, Debug)]
 pub struct NewCloneService<S>(S);
 
+#[derive(Debug)]
+pub struct NewFromTargets<P, N> {
+    inner: N,
+    _marker: PhantomData<fn() -> P>,
+}
+
+/// Builds services by passing `P` typed values to an `N`-typed inner stack.
```

Comment:
this doesn't mention that `P` is constructed from a `(T, U)` tuple...should the comment state something about that?